resteasy-spring-boot-starter/src/main/java/com/paypal/springboot/resteasy/ResteasySpringBootConfig.java

```diff
@@ -35,17 +35,15 @@
 
 	private static Logger logger = LoggerFactory.getLogger(ResteasySpringBootConfig.class);
 
-	private static ResteasyProviderFactory resteasyProviderFactory = new ResteasyProviderFactory();
```

Comment:
Yes, I had not read the issue in stack overflow, so I take that back. Now I got what is going on. Thanks a lot for reporting it quickly and for the PR too. I am going to merge it and release a new version today.